apps/client/src/common/components/view-params-editor/constants.ts

```diff
@@ -263,6 +263,126 @@ export const LOWER_THIRDS_OPTIONS: ParamField[] = [
   },
 ];
 
+export const LOWER_THIRD_OPTIONS: ParamField[] = [
+  {
+    id: 'upper-src',
+    title: 'Upper Text',
+    description: '',
+    type: 'option',
+    values: {
+      title: 'Title',
+      subtitle: 'Subtitle',
+      presenter: 'Presenter',
+      notes: 'Notes',
+      lower: 'Lower Third',
+    },
+  },
+  {
+    id: 'lower-src',
+    title: 'Lower Text',
+    description: '',
+    type: 'option',
+    values: {
+      subtitle: 'Subtitle',
+      title: 'Title',
+      presenter: 'Presenter',
+      notes: 'Notes',
+      lower: 'Lower Third',
+    },
+  },
+  {
+    id: 'upper-colour',
+    title: 'Upper Text Colour',
+    description: 'Upper text colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: '0000ff (default)',
+  },
+  {
+    id: 'lower-colour',
+    title: 'Lower Text Colour',
+    description: 'Lower text colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: '0000ff (default)',
+  },
+  {
+    id: 'upper-bg',
+    title: 'Upper Background Colour',
+    description: 'Upper text background colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: 'dddddd44 (default)',
+  },
+  {
+    id: 'lower-bg',
+    title: 'Lower Background Colour',
+    description: 'Lower text background colour in hexadecimal',
+    prefix: '#',
+    type: 'string',
+    placeholder: 'dddddd44 (default)',
+  },
+  {
+    id: 'upper-font',
+    title: 'Upper Text Font',
+    description: 'https://developer.mozilla.org/en-US/docs/Web/CSS/font',
+    type: 'string',
+    placeholder: 'bold 4vh sans-serif',
+  },
+  {
+    id: 'lower-font',
+    title: 'Lower Text Font',
+    description: 'https://developer.mozilla.org/en-US/docs/Web/CSS/font',
+    type: 'string',
+    placeholder: 'normal 3.5vh sans-serif',
+  },
+  {
+    id: 'width',
+    title: 'Width',
+    description: 'Width of the element',
```

Comment:
is this the width relative to the screen?
Should this not be automated related on the amount of text?

Reply:
For my lowers I like that the element always reaches the same position on the screen even if the name is short
but maybe we cover more users with a dynamic one?
I could change it to be minimum width?